### DISCUSSION TITLE
v0.14.1 is released

### DISCUSSION THREAD

**Discussion by tjex** [zk-org/zk#418] [General]
Listed below are the major fixes and updates for v0.14.1. 
However, many "behind the scenes" fixes and maintenance work from new maintainers @mcDevnagh, @julio-lopez and @jurica in particular were integral to this release as it marks the first release after having transferred the project from @mickael-menu's solo hands to [zk-org](https://github.com/zk-org). Thanks of course also goes to Mickael for his insight and allowing us to pester him a little bit more as we get to grips with the code base.

We also now how builds for Alpine Linux 🎉

See you next time,
@tjex 

## What's Changed

* Fixed parsing large notes @khimaros in https://github.com/zk-org/zk/pull/339
* fix day range parsing (zk-org/zk#382) by @tjex in https://github.com/zk-org/zk/pull/384
* accept tripple dash file URIs as valid links by @tjex in https://github.com/zk-org/zk/pull/391
* fix(lsp): fix trigger completion of zk LSP by @Rahlir in https://github.com/zk-org/zk/pull/397
* fix(lsp): ignore diagnostic check within code blocks by @Rahlir in https://github.com/zk-org/zk/pull/399
* allow notebook as hidden dir by @tjex in https://github.com/zk-org/zk/pull/402

## Documentation / Website

* Update documentation to restore GitHub Pages functionality by @hugginsio in https://github.com/zk-org/zk/pull/387
* Fix hyperlink from `README.md` to `docs/getting-started.md` by @sumit-sahrawat in https://github.com/zk-org/zk/pull/395
* Doc: how to edit today's daily note by @apraga in https://github.com/zk-org/zk/pull/407

## New Contributors
* @khimaros made their first contribution in https://github.com/zk-org/zk/pull/339
* @julio-lopez made their first contribution in https://github.com/zk-org/zk/pull/354
* @mcDevnagh made their first contribution in https://github.com/zk-org/zk/pull/374
* @tjex made their first contribution in https://github.com/zk-org/zk/pull/376
* @hugginsio made their first contribution in https://github.com/zk-org/zk/pull/387
* @sumit-sahrawat made their first contribution in https://github.com/zk-org/zk/pull/395
* @Rahlir made their first contribution in https://github.com/zk-org/zk/pull/397
* @apraga made their first contribution in https://github.com/zk-org/zk/pull/407

**Full Changelog**: https://github.com/zk-org/zk/compare/v0.14.0...v0.14.1

<hr /><em>This discussion was created from the release <a href='https://github.com/zk-org/zk/releases/tag/v0.14.1'>v0.14.1</a>.</em>

**Comment by jurica** [comment]
Very cool! Thanks @tjex for all the work you put into it!
